### ISSUE TITLE
Action Date Field is broken when change month

### ISSUE THREAD

**Issue by mmanzano** [laravel/nova-issues#760]
I'm trying to add a Date Field for an Action. When I go to next or previous month the modal disappear. When I select a date I can see a console error:

```
app.js?id=e827b529fdcb55c987a8:1 Uncaught TypeError: Cannot read property 'value' of undefined
    at a.onChange (app.js?id=e827b529fdcb55c987a8:1)
    at ut (vendor.js?id=85c7b1490fbdaf9eab57:1)
    at Object.c.close (vendor.js?id=85c7b1490fbdaf9eab57:1)
    at it (vendor.js?id=85c7b1490fbdaf9eab57:1)
    at ot (vendor.js?id=85c7b1490fbdaf9eab57:1)
    at HTMLDivElement.<anonymous> (vendor.js?id=85c7b1490fbdaf9eab57:1)
```

I launch the action:

![image](https://user-images.githubusercontent.com/1055699/46340462-c24a0f80-c635-11e8-9b4b-53d981c228f8.png)

When I click previous:

![image](https://user-images.githubusercontent.com/1055699/46340404-9cbd0600-c635-11e8-8b78-925420800a7f.png)

**Comment by jonsugar** [comment]
I can confirm this issue exists for both Date and DateTime fields. 

I can only pick dates that on the initial popup that appears. Clicking previous, next or the year will hide the containing modal leaving the date picker.

**Comment by seanmangar** [comment]
I have the same issue, any idea when it'll be fixed?

**Comment by disitec** [comment]
We had a similar issue in our date filter package. Maybe our fix could help

https://github.com/64robots/nova-date-filter/issues/1

**Comment by demency** [comment]
Related to #887.

I solved the error by eliminating the modal-close event on line 6 of the ConfirmActionModal component. Apparently it detected that the click was outside the modal, therefore the closing event was emitted. After recompiling the assets files the modal was not closed and the input works correctly.

**Comment by seanmangar** [comment]
Awesome, thanks @davidhemphill!

**Comment by shradhabanerjee** [comment]
Hi @davidhemphill , 

> Nova Version : 2.0.11

> Laravel Version : 5.8

> PHP Version : 7.2.20

This issue is still present in chrome browser. It works fine on safari and Firefox. Is there any fix available for the same on chrome ? I tried the resolution given by @demency , doesn't help on chrome.
Updated Nova, Laravel. Did nova:publish --force.

Attaching the screenshot of the same.

<img width="490" alt="Screenshot 2019-08-28 at 11 31 14 PM" src="https://user-images.githubusercontent.com/46675689/63880861-2911ad80-c9ec-11e9-826d-42e7c6c2163f.png">
